### PR TITLE
Replace mailing list / IRC mention with link to Community page

### PR DESCRIPTION
**Pull request by juliusv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ The Makefile provides several targets:
 
   * The source code is periodically indexed: [Prometheus Core](http://godoc.org/github.com/prometheus/prometheus).
   * You will find a Travis CI configuration in `.travis.yml`.
-  * All of the core developers are accessible via the [Prometheus Developers Mailinglist](https://groups.google.com/forum/?fromgroups#!forum/prometheus-developers) and the `#prometheus` channel on `irc.freenode.net`.
+  * You can reach the Prometheus developers and users on various communication channels. For details, see the [Community page](https://prometheus.io/community).
```
Comment (brian-brazil) [inline README.md]: How about "See the community page for how to reach..."?
It's a little shorter and mentions the link earlier

Comment (RichiH) [inline README.md]: Agreed with @brian-brazil

**Comment by fabxc** [comment]
Fix, discussion, or close please.

**Comment by RichiH** [comment]
@juliusv Can you update your patch, or just merge this one?

**Comment by juliusv** [comment]
Updated.

**Comment by RichiH** [comment]
Thanks :)

**Pull request merged**